### PR TITLE
migrates tests to new test repository

### PR DESCRIPTION
**Pull request by nyuuyn**
Migrates tests to new [test repository](https://github.com/OpenTOSCA/tosca-definitions-test-applications)

### CODE REVIEW

**Review by zimmerml — APPROVED** [review]

**Review by wederbn — APPROVED** [review]
LGTM, only some minor comments

File: org.opentosca.container.war/src/test/java/org/opentosca/container/war/tests/TestUtils.java [hunk 1]
```diff
@@ -101,48 +101,60 @@ public static Csar fetchCSARFromRepository(RepositoryConfigurationObject.Reposit
...
-                cloneRepo(repositoryPath, remoteUrl);
             }
         }
 
         // inject the current path to the repository factory
-        RepositoryFactory.reconfigure(
-            new GitBasedRepositoryConfiguration(false, new FileBasedRepositoryConfiguration(repositoryPath, provider))
-        );
-        IRepository repository = RepositoryFactory.getRepository();
+        IRepository repository;
```
[the remark this answers is not in the document]
Comment (nyuuyn) [reply org.opentosca.container.war/src/test/java/org/opentosca/container/war/tests/TestUtils.java]: fixed

File: org.opentosca.container.war/src/test/java/org/opentosca/container/war/tests/AdaptMultiMyTinyToDoIntegrationTest.java [hunk 2]
```diff
@@ -38,7 +38,9 @@
 @TestPropertySource(properties = "server.port=1337")
 public class AdaptMultiMyTinyToDoIntegrationTest {
 
-    public QName csarId = new QName("http://opentosca.org/servicetemplates", "Multi_MyTinyToDo_Bare_Docker_w1-wip1");
+    public static final String TestApplicationsRepository = "https://github.com/OpenTOSCA/tosca-definitions-test-applications";
```
[the remark this answers is not in the document]
Comment (nyuuyn) [reply org.opentosca.container.war/src/test/java/org/opentosca/container/war/tests/AdaptMultiMyTinyToDoIntegrationTest.java]: fixed

File: .github/workflows/maven.yml [hunk 3]
```diff
@@ -25,15 +25,15 @@ jobs:
     - name: Check out TOSCA internal repository
       uses: actions/checkout@v2
       with:
-        repository: OpenTOSCA/tosca-definitions-internal
-        ref: 'master'
+        repository: OpenTOSCA/tosca-definitions-test-applications
+        ref: 'main'
         token: '${{secrets.OPENTOSCAPRIVATEREPOACCESS}}'
```
[the remark this answers is not in the document]
Comment (nyuuyn) [reply .github/workflows/maven.yml]: fixed

File: org.opentosca.container.war/src/test/java/org/opentosca/container/war/tests/MigrateMyTinyToDo2MultiMyTinyToDoIntegrationTest.java [hunk 4]
```diff
@@ -43,10 +43,12 @@
 @TestPropertySource(properties = "server.port=1337")
 public class MigrateMyTinyToDo2MultiMyTinyToDoIntegrationTest {
 
+    public static final String TestApplicationsRepository = "https://github.com/OpenTOSCA/tosca-definitions-test-applications";
```
[the remark this answers is not in the document]
Comment (wederbn) [reply org.opentosca.container.war/src/test/java/org/opentosca/container/war/tests/MigrateMyTinyToDo2MultiMyTinyToDoIntegrationTest.java]: Somehow I thought of TEST_APPLICATIONS_REPOSITORY :D

**Pull request merged**